### PR TITLE
Surveys: Add Started by and startTime [WIP]

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

File: wavelength-plugins/survey.js [hunk 1]
```diff
@@ -387,14 +393,14 @@ exports.commands = {
...
-		"Accepts the following commands:",
-		"/survey create [question] - Create a survey. Allows up to 5 surveys. Requires % @ # & ~",
-		"/survey answer [survey number], [answer] - Answers the specified survey.",
-		"/survey results [survey number] - View the results of the specified survey. You can't go back and answer if you haven't already.",
-		"/survey display (survey number) - Display the specified survey. If no ID is specified, displays all surveys.",
-		"/survey remove [user], [survey number] - Removes a user's reply from the specified survey and prevents them from sending in a new one for this survey. Requires: % @ # & ~",
-		"/survey end [survey number] - Ends the specified survey and displays the results. Requires: % @ # & ~",
-		"/survey timer [minutes | clear], [survey number] - Sets a timer for the specified survey to automatically end. Require % @ # & ~",
+		`/survey allows rooms to run their own surveys. These surveys are limited to five surveys at a time per room.
+		Accepts the following commands
```
Comment (Mystifi) [inline wavelength-plugins/survey.js]: This should still have a semicolon after it.

File: wavelength-plugins/survey.js [hunk 2]
```diff
@@ -387,14 +393,14 @@ exports.commands = {
...
+		/survey create [question] - Create a survey. Allows up to 5 surveys. Requires % @ # &.
```
Comment (Mystifi) [inline wavelength-plugins/survey.js]: Should there be a note that talks about `survey htmlcreate [question] requiring leaders and above? Also, why did you remove the ~ from this line but not from `survey end`?

File: wavelength-plugins/survey.js (hunk 1 above)
Comment (DeathlyPlays) [reply wavelength-plugins/survey.js]: Colon (:) or Semi (;)

File: wavelength-plugins/survey.js (hunk 2 above)
Comment (DeathlyPlays) [reply wavelength-plugins/survey.js]: Honestly idk

**Comment by Volco** [comment]
@DeathlyPlays this works correct?

**Comment by DeathlyPlays** [comment]
Well honestly sometimes Started by works sometimes not so much idk why sometimes it works others not at all

**Comment by Volco** [comment]
I'll take a look then when i can (i'll be testing it is what i mean)

**Comment by Volco** [comment]
@DeathlyPlays other than that is it all good? (fixed the started by)

**Comment by DeathlyPlays** [comment]
That should be it
